pages/dkp/kommander/2.2/workspaces/applications/platform-applications/index.md

```diff
@@ -35,16 +34,16 @@ To use the CLI to deploy or uninstall applications, see [Application Deployment]
 | external-dns-2.20.5           | external-dns          | False               |
 | fluent-bit-0.16.2             | fluent-bit            | False               |
 | gatekeeper-0.6.8              | gatekeeper            | False               |
-| grafana-logging-6.16.14        | grafana-logging       | False               |
```

Comment:
These version changes might need to stick around from this PR https://github.com/mesosphere/dcos-docs-site/pull/3939/files. Looking at kommander-applications, it looks like these previous versions should be correct.

But honestly, I think we will revisit these again before 2.2 is released since I suspect more versions will be changed before 2.2 😅

Reply:
yeah agree here. not a deal breaker as I'm sure we'll update these version numbers going forward.